python/syncinstances/sync_instances.py

```diff
@@ -0,0 +1,94 @@
+import sys
+
+import boto3
+
+src_region = "us-east-1"
+tgt_region = "us-west-2"
```

Comment:
It would be nice if these and the instance ids were inputs to the script

Reply:
good call, this was me being lazy.